frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -714,7 +734,7 @@ pub mod pallet {
 				 deprive validator from their authoring reward.";
 
 			// Check score being an improvement, phase, and desired targets.
-			Self::unsigned_pre_dispatch_checks(&solution).expect(error_message);
+			Self::unsigned_pre_dispatch_checks(&solution, round).expect(error_message);
```

Comment:
I don't have any objection to this as well, but if convenient and no sensible, did you consider putting `round` inside `RawSolution`?

Reply:
Um. Good catch! It's already there. Fixing.

Reply:
89cf445cd